lib/git_reflow.rb

```diff
@@ -127,32 +123,36 @@ def deliver(options = {})
         if existing_pull_request.good_to_merge?(force: options['skip_lgtm'])
           puts "Merging pull request ##{existing_pull_request.number}: '#{existing_pull_request.title}', from '#{existing_pull_request.feature_branch_name}' into '#{existing_pull_request.base_branch_name}'"
 
-          update_destination(base_branch)
-          merge_feature_branch(feature_branch,
-                               :destination_branch  => base_branch,
-                               :pull_request_number => existing_pull_request.number,
-                               :lgtm_authors        => existing_pull_request.approvals,
-                               :message             => commit_message)
-          committed = run_command_with_label 'git commit', with_system: true
-
-          if committed
-            say "Merge complete!", :success
+          begin
+            res = existing_pull_request.merge_feature_branch(
+              feature_branch,
+              :destination_branch  => base_branch,
+              :pull_request_number => existing_pull_request.number,
+              :lgtm_authors        => existing_pull_request.approvals,
+              :title               => existing_pull_request.title,
+              :message             => commit_message,
+              :head                => existing_pull_request.head
+            )
+          rescue StandardError => error
```

Comment:
Change this to `rescue GitReflow::GitServer::PullRequest::MergeError`.  Rescuing from `StandardError` is typically a code-smell that points to fuzzy logic.